### ISSUE TITLE
List supported agencies in the sidebar

### ISSUE THREAD

**Issue by hathix** [trynmaps/metrics-mvp#456]
Or perhaps elsewhere in the UI. This will help as we scale up.

I'm not sure if the TriMet data is current visible or if it just resides in someone's branch. Whenever users become able to view other agencies' data, we should take action on this issue.

**Comment by youngj** [comment]
I don't think it has actually been decided whether we want to have one web app that shows data for all cities, or have separate web apps for different cities. 

I was thinking Portland and SF would be separate web apps on different domains, and each web app could show multiple agencies within its region. 

I think that having separate web apps for different cities would be better in the long run because it would make it possible to be less centralized, rather than forcing everything to go through one team and one codebase. The backend is designed so that teams in different cities could relatively easily set up their own AWS account, S3 bucket, and Kubernetes cluster rather than using any centralized infrastructure from the OpenTransit team.

There could be a centralized directory that links to the apps for various cities.

**Comment by hathix** [comment]
You're right that a federated model would reduce reliance on the central OpenTransit team. (Still, it's worth offering cities an option to host their instance on the OT website, since not everyone will want to pay for their own domain.)

Regardless, if even one city wants to use their own domain (quite likely), we'd have to store a separate registry of OT instances. Perhaps we could just make a config file in this repo and city leads could make a PR adding their city's url to the list.

**Comment by hathix** [comment]
The question now becomes how best to implement the list of agencies- a question best answered once the pdx team has set up its instance publicly.
